### PR TITLE
Finish migrating indexing docs from google doc

### LINKED ISSUE #897: Move indexing documentation into this repository
@kayiwa I would like to move the indexing documentation (Steps to create a new full index) in this repository? Do you see any sensitive information in the documentation https://docs.google.com/document/d/1bHvgfgyUmDXV7ROqEZFaRxJFEYhSSUbeQoQT883GMWg/edit#heading=h.52h6kcl0n8c4 that would prevent me to do so?

### PR DESCRIPTION
**Pull request by hackartisan**
This gets everything migrated; we'll of course have to keep updating this documentation as things change.

Many parts of that google doc are no longer relevant since we're moving off of voyager.

Note that I didn't migrate the recommendation to use the catalog-staging instance for observing the indexing process. It could introduce inconsistencies with regard to the numismatics items via sneakers. Instructions for hooking up a local dev instance to the index were migrated for observing via the orangelight UI.

closes #897

### CODE REVIEW

**Review by tpendragon — APPROVED** [review]

File: docs/indexing.md [hunk 1]
```diff
@@ -1,69 +1,96 @@
...
 
 These files are on an aws s3 bucket; the password is in lastpass under SCS Prod S3 Keys in the shared `bibdata` directory. The file path is SCSB > data-exports > PUL > MARCXml > Full. There are .zip files there and .csv files. The zip files contain the MARC dumps.
 
 Frequency of updates: once per day
 
 SCSB ids start with ‘SCSB’
 
-## Source: Theses
+## Source: DSpace
 About 68,800 senior theses pulled from the OIT Dataspace repository http://dataspace.princeton.edu/jspui/handle/88435/dsp019c67wm88m
```
Comment (escowles) [inline docs/indexing.md:24]: Probably good to remove OIT here.

File: docs/indexing.md [hunk 2]
```diff
@@ -75,27 +102,86 @@ in `/tmp/updates/` and as they are processed they will be moved to `/data/marc_l
...
-- `$ SET_URL=http://lib-solr8-staging.princeton.edu:8983/solr/catalog-alma-staging-rebuild bundle exec rake numismatics:index:full 2> /tmp/numismatics_index_[date].log`
+- `$ SET_URL=http://lib-solr8-prod.princeton.edu:8983/solr/catalog-alma-rebuild bundle exec rake numismatics:index:full 2> /tmp/numismatics_index_[date].log`
 - It will show a progress bar as it runs. Takes 30 min to an hour.
 - Note that the default log writes to STDERR to distinguish its output from the progress bar
+
+### Logs
+
+Many of these tasks output log messages.
+
+Because traject does not log to datadog we lose log output for indexing that runs in the background. See https://github.com/pulibrary/bibdata#1507
```
Comment (escowles) [inline docs/indexing.md:141]: The link doesn't work for me, so I think it should be https://github.com/pulibrary/bibdata/issues/1507

File: docs/call-numbers.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Call Numbers
+
+Note: This documentation was migrated from https://docs.google.com/document/d/1bHvgfgyUmDXV7ROqEZFaRxJFEYhSSUbeQoQT883GMWg/edit#heading=h.oz6ax25g11u and needs to be updated
+
+## Generating browse lists
+
```
Comment (christinach) [inline docs/call-numbers.md:6]: ssh to catalog-qa1 for staging or catalog.. for production.

File: docs/indexing.md [hunk 4]
```diff
@@ -75,27 +102,86 @@ in `/tmp/updates/` and as they are processed they will be moved to `/data/marc_l
...
+- `> solr.commit`
+
+You can see the number of records now in the solr admin panel.
+
+Query to get the SCSB:
+- These are only the records that someone can get in the advanced search -> holding location -> scsbcul or scsbnypl.
+- `> response_scsb = solr.get ‘select’, :params=> {:q => ‘id:SCSB*’}`
+
+Query to delete SCSB:
+- `> solr.delete_by_query("id:SCSB*")`
```
Comment (christinach) [inline docs/indexing.md:183]: This is ok to keep.

File: docs/indexing.md [hunk 5]
```diff
@@ -75,27 +102,86 @@ in `/tmp/updates/` and as they are processed they will be moved to `/data/marc_l
...
+```
+PRODUCTION=catalog-production2 REBUILD=catalog-production1 bundle exec cap solr8-production alias:swap
+```
+
+To turn sneakers workers back on:
+- cd in your local princeton_ansible directory -> pipenv shell -> ansible orangelight_alma_prod -u pulsys -m shell -a "sudo service orangelight-sneakers start"
+
+## Other tasks
+
+### Delete records from voyager, but leave scsb / thesis records
```
Comment (christinach) [inline docs/indexing.md:167]: Can we update to `Delete records from Solr, excluding SCSB and Theses`

File: docs/indexing.md [hunk 6]
```diff
@@ -75,27 +102,86 @@ in `/tmp/updates/` and as they are processed they will be moved to `/data/marc_l
...
+Tunnel to the solr box, go to admin panel, you can see how many records there are by submitting a blank query
+- Ssh to marc_liberation box as deploy user
+- `$ bin/rails c`
+- `> solr_url = "http://lib-solr.princeton.edu:8983/solr/catalog-rebuild"`
```
Comment (christinach) [inline docs/indexing.md:171]: `solr_url = "http://lib-solr8-prod.princeton.edu:8983/solr/catalog-rebuild"`

File: docs/call-numbers.md (hunk 3 above)
Comment (christinach) [reply docs/call-numbers.md:6]: navigate to /opt/orangelight/current

**Review by christinach — CHANGES_REQUESTED** [review]

File: docs/call-numbers.md (hunk 3 above)
Comment (christinach) [reply docs/call-numbers.md:6]: To fully regenerate all browse lists:
-  on catalog-staging `ssh to a staging box`, cd `/opt/orangelight/current` and run: ... (There are no workers for catalog-staging and the rake task needs to run on the catalog box.)
- for catalog production `ssh to catalog-indexer1` (confirm through the crontab that it's running the call numbers and not other rake tasks), cd `/opt/orangelight/current` and run: ...

File: docs/call-numbers.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+To fully regenerate all browse lists:
+`OL_DB_PORT=5432 bundle exec rake browse:all && OL_DB_PORT=5432 bundle exec rake browse:load_all`
+
+Expected time: 5.5 - 6 hours.
+
+## Troubleshooting
+
+When browse lists fail to generate and would like to run the rake task before it is scheduled:
+ssh to `lib-pulindexer1`. Check the crontab and make sure the rake task for the call numbers runs on this box. If not then check `lib-pulindexer2` and `lib-pulindexer3`.
```
Comment (christinach) [inline docs/call-numbers.md:15]: catalog-indexer1, catalog-indexer2, catalog-indexer3. Currently the call_numbers task is deployed to run on catalog-indexer1 check the crontab to confirm.

**Review by christinach — CHANGES_REQUESTED** [review]

**Comment by hackartisan** [comment]
Thank you for the careful review. I've made the recommended changes; ready for re-review

File: docs/call-numbers.md (hunk 7 above)
Comment (christinach) [reply docs/call-numbers.md:22]: @hackartisan can you update the worker names? ^

**Review by christinach — CHANGES_REQUESTED** [review]

**Comment by hackartisan** [comment]
@christinach fixed, thank you.

**Review by christinach — APPROVED** [review]
Thank you @hackartisan

**Pull request merged**